api.c

```diff
@@ -801,7 +801,7 @@ void getAllQueries(char *client_message, int *sock)
 
 		if(istelnet[*sock])
 		{
-			ssend(*sock,"%i %s %s %s %i %i %i %lu\n",queries[i].timestamp,qtype,domain,client,queries[i].status,queries[i].dnssec,queries[i].reply,delay);
+			ssend(*sock,"%i %s %s %s %i %i %i %lu %i\n",queries[i].timestamp,qtype,domain,client,queries[i].status,queries[i].dnssec,queries[i].reply,delay,i);
```

Comment:
Is this intended?

Reply:
Yes. It is harmless for the PHP API as this number will just be appended to the individual JSON arrays. However, as this is a pure debugging property, I will wrap it in an `if(debug)` statement.